components/Start/StartButton/index.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function StartButton() {
+  return (
+    <StartLink href="/quiz" alt="StartLink">
+      Start
+    </StartLink>
+  );
+}
+
+export const StartLink = styled(Link)`
+  display: block;
+  justify-content: center;
+  align-items: center;
```

Comment:
As far as I know is `justify-content and align-items` only working along if the container is set to `display: flex (& grid)` but not sure though 🙂